lib/database/models/run.js

```diff
@@ -104,6 +104,24 @@ module.exports = (sequelize) => {
         aliceDipolePolarity: {
             type: Sequelize.CHAR(32),
         },
+        trgGlobalRunEnabled: {
+            type: Sequelize.BOOLEAN,
+        },
+        trgEnabled: {
+            type: Sequelize.BOOLEAN,
+        },
```

Comment:
You put default null in migration but not here, is it normal?

Reply:
Yes, this is for the already existing values. Within the dto, it is already handled to be false when there are new values put in (because again of the go API limits).